pkg/controller/plan/adapter/vsphere/vsphere_suite_test.go

```diff
@@ -11,3 +12,26 @@ func TestVsphere(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "vSphere Suite")
 }
+
+func TestGetDeviceNumber(t *testing.T) {
```

Comment:
this test should rather be in the 'util' package but ok, we can move and improve it separately